enhancements/plain-k8s-bundles.md

```diff
@@ -0,0 +1,220 @@
+---
+title: plain-k8s-bundles
+authors:
+  - "@exdx"
+  - "@njhale"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-03-09
+last-updated: 2020-03-09
+status: implementable
+see-also:
+  - "https://docs.google.com/document/d/1kDDVVdjSjBAP3-4QnIPb67QfcR2-LEWRbvKFUDzEwrc/edit#heading=h.70izt81ko44h"
+---
+
+# plain-k8s-bundles
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Add support for a new mediatype, `k8s+v1`, to the [Operator Bundle spec](https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md), [operator-registry](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go), and [OLM](https://github.com/operator-framework/operator-lifecycle-manager) that enables packaging an operator without defining a [ClusterServiceVersion](https://github.com/operator-framework/olm-book).
+
+## Motivation
+
+Operator authors find CSVs difficult to write and maintain. Expressing an OLM-managed operator as a set of related kubernetes objects is more intuitive to users. Supporting plain kubernetes manifests makes OLM more approachable to the wider community, which is relevant in light of the Operator Framework attempting to become a CNCF project. This feature is part of the overall goal of making OLM simpler to use.
+
+Furthermore, this feature is part of the overall goal of making OLM more flexible by supporting more media types. The more media types OLM supports the more popular and accesible OLM becomes.
+
+### Goals
```

Comment:
Could an operator that was shipped as a CSV move to the CSVless option?

Reply:
Yes, but not in an automated way -- are you suggesting this as a goal?

Reply:
Depends on the persona. An operator author would clearly need to create the new resources themselves but a Cluster Admin should be able to upgrade from a previous iteration of the operator implemented with the CSV to a new version using a CSVless option with no leftover resources.

Reply:
Ah -- CSVless bundles will still result in a CSV on-cluster, so there should be no clean up for a Cluster Admin to do. Does that help, or am I looking past something?